tf2onnx/optimizer/__init__.py

```diff
@@ -6,4 +6,4 @@
 from __future__ import print_function
 from __future__ import unicode_literals
 
-__all__ = ["transpose_optimizer"]
+__all__ = ["identity_optimizer", "transpose_optimizer"]
```

Comment:
consider export IdentityOptimizer and TransposeOptimizer directly?
so user can use ```from tf2onnx.optimizer import IdentityOptimizer ``` to import specific optimizer or ```from tf2onnx.optimizer import *``` to import all.

```
from tf2onnx.optimizer.identity_optimizer import IdentityOptimizer
from tf2onnx.optimizer.transpose_optimizer import TransposeOptimizer
```
 feels too verbose

Reply:
sounds good.